README.md

```diff
@@ -1,53 +1,68 @@
-Gaffer
-======
+![Gaffer Logo](resources/GafferLogo.svg)
 
-Gaffer is an open source application framework designed specifically for creating tools for use in visual effects production. It builds on top of the Cortex libraries, adding a flexible node-based computation framework and a user interface framework for editing and viewing node graphs. Gaffer ships with a number of sample modules and applications, not least of which is a module for the on-demand generation of procedural scenes for rendering.
+# Gaffer #
 
-More information can be found on the [project homepage](http://gafferhq.org).
+Gaffer is an open source application framework designed for creating tools for use in VFX production. It builds on top of the Cortex libraries, adding a flexible node-based computation framework and a user interface framework for editing and viewing node graphs. Gaffer ships with a number of sample modules and applications, not least of which is a module for the on-demand generation of procedural scenes for rendering.
```

Comment:
I think Gaffer has matured enough now that we can bill it as an application first and a framework second, rather than as a framework and a sample application. Could we use something more like the copy from gafferhq.org?